src/components/NavBar.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import '../styles/NavBar.css';
+import PropTypes from 'prop-types';
+
+const NavBar = ({buttons, menuOptions}) => {
+    const [navStatus, useNavBar] = useState(false);
+    return (
+        <>
+            <header className="header">
+                <div className="navContainer">
+                    <span
+                        className="logo"
+                        style={{
+                            color: '#fff',
+                            fontStyle: 'italic',
+                            fontWeight: '400',
+                        }}
+                    >
+                        BidsDirect: Find The Best Fit For Your Next Project
+                    </span>
+                    <nav>
+                        {buttons.map((button, key = button.key) => <div key={key}>{button.html}</div>)}
```

Comment:
dont pass the html in the object...just the text...put the jsx directly in here insteal

Reply:
im not seeing how i can dynamically do this and satisfy the key requirement.